prisma/migrations/20241124172312_rename_column/migration.sql

```diff
@@ -0,0 +1,17 @@
+/*
+  Warnings:
+
+  - You are about to drop the column `shortUrl` on the `link` table. All the data in the column will be lost.
```

Comment:
**issue**: Dropping the `shortUrl` column will result in data loss. Ensure that this is intentional and that data is backed up if necessary.